draft-packages/hero-card/docs/HeroCard.stories.tsx

```diff
@@ -14,7 +14,7 @@ const renderContent = () => (
       height: "300px",
     }}
   >
-    <p style={{ margin: "0" }}>
+    <p style={{ margin: "0 0 0.75rem 0" }}>
```

Comment:
Interesting, why did the margin change here?

Reply:
I added a minor change... couldn't abide by the story not actually having any spacing between a typographical element and a button 😆

Reply:
For context a before and after:
![Screen Shot 2022-11-15 at 2 44 13 pm](https://user-images.githubusercontent.com/36558508/201821411-7f805458-74ff-45ad-8ce9-b9955144586c.png)